release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,82 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+[id="ocp-4-14-32_{context}"]
+=== RHSA-2024:4329 - {product-title} 4.14.32 bug fix and security updates
+
+Issued: 2024-07-11
+
+{product-title} release 4.14.32, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4329[RHSA-2024:4329] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:4332[RHBA-2024:4332] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.32 --pullspecs
+----
+
+[id="ocp-4-14-32-enhancements_{context}"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-14-32-pipeline-trigger_{context}"]
+===== New custom resource definitions for pipeline plugin
+
+This release updates the pipelines plugin to support the latest Pipeline Trigger API version for the custom resource definitions (CRDs) `ClusterTriggerBinding`, `TriggerTemplate`, and `EventListener`. (link:https://issues.redhat.com/browse/OCPBUGS-35723[*OCPBUGS-35723*])
+
+[id="ocp-4-14-32-_{context}"]
+===== Controller for defragmentating etcd
+
+This release introduces a controller to defragment etcd for hosted clusters on Hypershift. (link:https://issues.redhat.com/browse/OCPBUGS-35290[*OCPBUGS-35723*])
+
+[id="ocp-4-14-32-insights-operator-resources_{context}"]
+===== Insights Operator collects new resources
+
+The Insights Operator (IO) now collects the `Prometheus` and `AlertManager` resources in addition to the `openshift-monitoring` custom resource. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
+
+[id="ocp-4-14-32-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the `alertmanager-trusted-ca-bundle` `ConfigMap` was not injected into the user-defined Alertmanager container, which prevented the verification of HTTPS web servers receiving alert notifications. With this update, the trusted CA bundle `ConfigMap` is mounted into the Alertmanager container at the `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` path. (link:https://issues.redhat.com/browse/OCPBUGS-36416[*OCPBUGS-36416*])
+
+* Previously, for clusters updated from older versions of {product-title} (OCP), enabling `kdump` on an OVN-enabled cluster sometimes prevented the node from rejoining the cluster or returning to the `Ready` state. This fix removes problematic stale data from older OCP versions and ensures this kind of stale data is always cleaned up. The node can now start correctly and rejoin the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-36356[*OCPBUGS-36356*])
```

Comment:
oh - just saw this as I was going to merge. "OPC" is not an approved abbreviation, should just be:
```suggestion
* Previously, for clusters updated from older versions of {product-title}, enabling `kdump` on an OVN-enabled cluster sometimes prevented the node from rejoining the cluster or returning to the `Ready` state. This fix removes problematic stale data from older OCP versions and ensures this kind of stale data is always cleaned up. The node can now start correctly and rejoin the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-36356[*OCPBUGS-36356*])
```